momepy/utils.py

```diff
@@ -229,7 +239,12 @@ def gdf_to_nx(
     fields = list(gdf_network.columns)
 
     if approach == "primal":
-        _generate_primal(net, gdf_network, fields, multigraph)
+        if oneway_column and ((not directed) and (not multigraph)):
```

Comment:
This could in theory work also for `networkx.DiGraph`, not only `MultiDiGraph`, right? Can we allow it?

Reply:
You are right, removed the restriction.